### ISSUE TITLE
[Feature] Make Slideshow options aware of clips

### ISSUE THREAD

**Issue by xx790** [stashapp/stash#3819]
**Is your feature request related to a problem? Please describe.**

Previously, images could only be static images, meaning slideshow delay is a simple matter.
Now we have clips, and clips have their own lengths that can vary.
Either we have to set the slideshow delay based on the longest clip or accept to see only the beginning of some clips.

**Describe the solution you'd like**

Option to view longer clips to completion - set delay to `max(options.delay, clip.duration)`.

**Describe alternatives you've considered**

Possible alternative: `options.delay` can be respected, but starting time for clips can be chosen randomly, so it is not only the beginning that is always played.
This seems less practical.

**Additional context**

Current options for reference:
![image](https://github.com/stashapp/stash/assets/118630824/f6ee3f0a-528d-484c-a6b0-f66e89b0f559)

Extra checkbox(es) can be added there to configure the behavior with clips.

(Related small issue: default value for the delay is set in milliseconds apparently and equals 5000, while it expects seconds and should've been 5.)

**Comment by Sassy0P** [comment]
I agree with this feature.
Also, should we use the same formula for animated gif's duration ?

**Comment by DogmaDragon** [comment]
> Also, should we use the same formula for animated gif's duration ?

Based on the scrope what Clips are meant to be, no.
> When a library has Videos disabled, Video Files (files ending with Video Extension) will be scanned as Image Clip.

**Comment by Sassy0P** [comment]
Gifs can also vary in length and if they are longer than the delay.
Then wouldn't a part of it will be skipped, like a clip would ?

I didn't mean to imply the exact same line but the idea of this formula in my last comment.

**Comment by DogmaDragon** [comment]
> Gifs can also vary in length and if they are longer than the delay. Then wouldn't a part of it will be skipped, like a clip would ?
> 
> I didn't mean to imply the exact same line but the idea of this formula in my last comment.

Right, but that would be a separate issue as .gif wouldn't be scanned as a clip in most cases.
